Update Shapes How-To for new Lasso tool (napari/#5555) (#149)
napari/docs: docs/howtos/layers/shapes.md

`Polygons` can be added using an array of N vertices. Polygons are closed by
default, and so you don't also need to include the first point at the end of the
array. The order of the vertices will determine the triangulation of the
polygon, which can be non-convex, but cannot have holes. For drawing polygons, 
multiple tools can be used.
The polygon creation tool can be selected from the layer control panel or by 
pressing the `P` key when the shapes layer is selected. When adding a polygon 
each click will add a vertex at the clicked location. To finish drawing a polygon 
you must click the`escape` key, which will add a final vertex at the current mouse 
position and complete the polygon. You'll then be able to start adding another one.

> `Polygons` can also be created with the polygon lasso creation tool. This can be found

Reviewer: ```suggestion
`Polygons` can also be created with the polygon lasso creation tool, which can be found
```

Author: Originally I had it the same, but wasn't certain about creating longer sentences.

in the layer control panel or by pressing `shift+P`. The tool can be used to draw 

> complex `Polygons` with the mouse

Reviewer: ```suggestion
complex `Polygons` with the mouse.
```

> For mouse draw mode the sequence of events by the user is as follows:

Reviewer: ```suggestion
The polygon lasso supports using the mouse or tablet. When using the mouse, the sequence of events to draw a polygon is as follows:
```

>     1. Mouse press
>     2. Move mouse while not mouse pressed
>     3. Mouse press to finish draw

Reviewer: ```suggestion
    1. Click mouse (left-click) to begin drawing.
    2. Move mouse—without holding down the mouse button—to draw the polygon.
    3. Click mouse (left-click) or press `escape` to end drawing—the polygon will auto-complete.
```

> `Polygons` can also be drawn using a tablet when using the polygon lasso tool. In this case, 

Reviewer: ```suggestion
The polygon lasso tool can also be used to draw `Polygons` using a tablet. In this case, 
```

> drawing is started by touching the tablet screen with the tablet pencil. The drawing will 
> be ongoing for as long as the pencil is moved while touching the tablet screen. For both 

Reviewer: ```suggestion
drawing the polygon is started by touching the tablet screen with the tablet stylus and drawing will continue for as long as the pencil is moved while touching the tablet screen. Note that similar behavior is also available when using a macOS trackpad, using three-finger drag mode.  
For both 
```

modes, vertices are added only if the vertex to be added is at least 10 screen pixels away 
from the previous vertex. As with the polygon creation tool drawing the shape can also be 
finished by pressing the `escape` key.

> When finishing a drawing using the polygon lasso tool an implementation of the Ramer–Douglas–Peucker 
> algorithm is applied to reduce the number of points that make up the shape, while conserving its

Reviewer: ```suggestion
After finishing drawing a polygon using the polygon lasso tool, an implementation of the [Ramer–Douglas–Peucker 
algorithm](https://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm) is applied to reduce the number of vertexes that make up the shape, while preserving its
```

> contours. The agressiveness by which the algorithm reduces the number of points of a shape is 

Reviewer: ```suggestion
contours. The aggressiveness with which the algorithm reduces the number of points of the polygon is 
```

> determined by an `epsilon` parameter. If 0 no points are removed. The higher the value of `epsilon`
> the more points get removed. The value of `epsilon` can be set by in napari going to 

Reviewer: ```suggestion
determined by an `epsilon` parameter, which is a perpendicular distance threshold. Any vertexes beyond the threshold will be preserved, so if `epsilon` is set to `0`, no vertexes will be removed. With increasing values of `epsilon`, more and more vertexes will be removed. The value of `epsilon` can be set by in napari going to 
```

`Polygons` can be added using an array of N vertices. Polygons are closed by
default, and so you don't also need to include the first point at the end of the
array. The order of the vertices will determine the triangulation of the
polygon, which can be non-convex, but cannot have holes. For drawing polygons, 
multiple tools can be used.
The polygon creation tool can be selected from the layer control panel or by 
pressing the `P` key when the shapes layer is selected. When adding a polygon 
each click will add a vertex at the clicked location. To finish drawing a polygon 
you must click the`escape` key, which will add a final vertex at the current mouse 
position and complete the polygon. You'll then be able to start adding another one.

`Polygons` can also be created with the polygon lasso creation tool, which can be 
found in the layer control panel or by pressing `shift+P`. The tool can be used to 

> draw complex `Polygons` with the mouse. The polygon lasso supports using the mouse 

Reviewer: ```suggestion
draw complex `Polygons` with the mouse
```

or tablet. When using the mouse, the sequence of events to draw a polygon is as 
follows:

    1. Click mouse (left-click) to begin drawing.
    2. Move mouse — without holding down the mouse button — to draw the polygon.

>     3. Click mouse (left-click) or press `escape` to end drawing—the polygon will auto-complete.

Reviewer: ```suggestion
1. Click mouse (left-click) to begin drawing.
2. Move mouse — without holding down the mouse button — to draw the polygon.
3. Click mouse (left-click) or press `escape` to end drawing—the polygon will auto-complete.
```

With the indentation this is showing as a code block and looks kinda bad:
<img width="839" alt="image" src="https://user-images.githubusercontent.com/76622105/236673742-22d8cc85-a000-45ed-b252-edb66cfd47f0.png">

The polygon lasso tool can also be used to draw `Polygons` using a tablet. In this case, 
drawing the polygon is started by touching the tablet screen with the tablet stylus and 
drawing will continue for as long as the pencil is moved while touching the tablet screen. 
Note that similar behavior is also available when using a macOS trackpad, using three-finger 
drag mode.  
For both modes, vertices are added only if the vertex to be added is at least 10 screen pixels 
away from the previous vertex. As with the polygon creation tool drawing the shape can also be 
finished by pressing the `escape` key.

After finishing drawing a polygon using the polygon lasso tool, an implementation of the [Ramer–Douglas–Peucker 
algorithm](https://en.wikipedia.org/wiki/Ramer–Douglas–Peucker_algorithm) is applied to reduce the 
number of vertices that make up the shape, while preserving its contours. The aggressiveness with 

> which the algorithm reduces the number of points of the polygon is determined by an `epsilon` parameter, 

Reviewer: ```suggestion
which the algorithm reduces the number of vertices of the polygon is determined by an `epsilon` parameter, 
```
